Sub/Src/HSV Filtering/hsv.py

```diff
@@ -0,0 +1,25 @@
+import cv2
+import numpy as np
```

Comment:
This script is not well organized and not modular. Please remove.
Aslo not in a valid folder. Anything vision related should be somewhere within Sub/Src/ComputerVision